doc/rtd/reference/custom_modules/custom_formats.rst

```diff
@@ -0,0 +1,51 @@
+.. _custom_formats:
```

Comment:
Wondering if there is some way to leave this content discoverable in format.rst and just provide a link in the custom_modules.rst that is created. I'd like to make sure minimally the `#part-handler` or `text/part-handler` mime part declaration type is searchable from the user-data format page as all user-data type declarations should probably be seen from that page.

Reply:
https://github.com/canonical/cloud-init/pull/5548#issuecomment-2252327342